plugins/woocommerce/includes/admin/class-wc-admin-notices.php

```diff
@@ -275,6 +312,7 @@ public static function update_notice() {
 	 * @deprecated 4.6.0
 	 */
 	public static function install_notice() {
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
Indeed, this looks like an attempt to convey some extra information in a way that isn't really supported by the function (but no real harm done since this parameter is only used for printing error messages).